Assets/Mirror/Core/Messages.cs

```diff
@@ -102,6 +103,38 @@ public struct EntityStateMessage : NetworkMessage
         public ArraySegment<byte> payload;
     }
 
+    // state update for unreliable sync.
+    // baseline is always sent over Reliable channel.
+    public struct EntityStateMessageUnreliableBaseline : NetworkMessage
```

Comment:
rename to `EntityStateMessageReliableBaseline` everywhere

Reply:
this is for Unreliable SyncMethod, I think it's more obvious this way

Reply:
Prove it's not sent on the reliable channel and I'll agree.

Reply:
the naming is based on SyncMethod, not on channel.
both ways are possible, but this is more obvious